lib/msf/core/handler/reverse_http/stageless.rb

```diff
@@ -26,7 +26,7 @@ def initialize_stageless
     ], self.class)
   end
 
-  def generate_stageless(&block)
+  def generate_stageless(ssl, &block)
```

Comment:
I'd prefer this be a keyword argument because it's rarely obvious from the call-site what's happening when you just pass a literal `true`.